docs/Tutorials/CCE.md

```diff
@@ -100,6 +103,32 @@ values are the complex modes in m-varies-fastest format. That is,
 "Lapse_Re(2,0)", "Lapse_Im(2,0)", "Lapse_Re(2,-1)", "Lapse_Im(2,-1)",
 "Lapse_Re(2,-2)", "Lapse_Im(2,-2)"
 ```
+
+#### Spherical harmonic nodes {#cartesian_spherical_nodes}
+
+\warning This format is not yet fully supported but is under development. If you
+need it please file an issue so we can escalate the priority.
+
+In this format the value of the functions at specially chosen collocation points
+(grid points) is read in. This allows SpECTRE to perform integrals, derivatives,
+and interpolation exactly on the input data. These grid points are
+Gauss-Legendre in $cos(\theta)$ and equally spaced in $\phi$. Below is a routine
+for computing the spherical harmonic $\theta$ and $\phi$ values. These can be
+used to compute the Cartesian locations for a given radius using the standard
+transformation. The routine supports \f$\ell\in[4, 32]\f$.
+
+\snippet Test_Spherepack.cpp spectre_cce_grid_point_locations
```

Comment:
Wrap this in

```html
<script>
function closeDetails() {
    document.getElementById("details").removeAttribute("open");
    window.location = "#details";
}
</script>

<details id="details">
<summary> SpECTRE CCE Gridpoint Locations </summary>
\snippet Test_Spherepack.cpp spectre_cce_grid_point_locations
<button onclick="closeDetails()">Close Details</button>
</details>
```

so there isn't just a giant wall of code in the middle of the tutorial. I haven't tried this, but if the button doesn't work just remove it and the `<script>`